autovalet_parking/src/autovalet_parking/Parker.py

```diff
@@ -86,40 +85,42 @@ def isReady(self):
         return self.ready
 
     def getParkingPoses(self):
-        return [self.goal1, self.goal2]
+        return self.goal
 
     def setParkDirection(self):
         tag_to_base_link = self.tf_buffer.lookup_transform(self.husky_frame_id,
                                                            self.aruco_frame_name,
                                                            rospy.Time(0),
                                                            rospy.Duration(1.0))
-        # print("center line", self.centerline_midpt)
-        # print("tag to base", tag_to_base_link)
         if self.centerline_midpt.y > tag_to_base_link.transform.translation.y:
             self.park_direction = "right"
         else:
             self.park_direction = "left"
-        # print("park_direction", self.park_direction)
 
     def calculateGoals(self):
         # if apriltag is to the right of the line
         if self.park_direction == "right":
-            pos1 = [0, 3, 1] # position relative to detected tag pose
-            rot1 = [0, -np.pi/4, -np.pi/4]
+            # position relative to detected tag pose
```

Comment:
add `(x,y,z)` to the end of this comment just for clarity

Reply:
as you wish